metricbeat/module/docker/helper.go

```diff
@@ -16,13 +16,29 @@ type Container struct {
 	Socket *string
 }
 
-func InitCurrentContainer(container *docker.APIContainers) *Container {
+func (c *Container) ToMapStr() common.MapStr {
+	m := common.MapStr{
+		"id":   c.Id,
+		"name": c.Name,
+		// TODO: Is this really needed
+		"socket": GetSocket(),
```

Comment:
I'm thinking in case a socket is used, if we should copy over the socket information to `metricset.host` info instead?